syntaxes/flip.tmLanguage.json

```diff
@@ -0,0 +1,94 @@
+{
+   "$schema": "https://raw.githubusercontent.com/martinring/tmlanguage/master/tmlanguage.json",
+   "name": "Flip",
+   "patterns": [
+      {
+         "include": "#keywords"
+      },
+      {
+         "include": "#storages"
+      },
+      {
+         "include": "#variables"
+      },
+      {
+         "include": "#strings"
+      },
+      {
+         "include": "#comments"
+      }
+   ],
+   "repository": {
+      "keywords": {
+         "patterns": [
+            {
+               "name": "keyword.control.flip",
+               "match": "\\b(return)\\b"
+            },
+            {
+               "name": "keyword.other.flip",
+               "match": "\\b(signal|typealias)\\b"
+            }
+         ]
+      },
+      "storages": {
+         "patterns": [
+            {
+               "name": "storage.type.flip",
+               "match": "\\b(model|class|enum|invariant)\\b"
+            },
+            {
+               "name": "storage.type.flip",
+               "match": "\\b(Array|Bool|Blob|Collection|Float|Int|Message|Map|ObjectRef|Optional|String|Variant|Vector)\\b"
+            },
+            {
+               "name": "storage.modifier.flip",
```

Comment:
Ok with that if VSCode users would expect that (or the same can be find in Java/Javascript, etc)